foundation/foundation-upgrade/src/main/resources/migrations.json

```
@@ -77,6 +88,34 @@
             "excludes": ["**/target/**", "**/templates/**"]
           }
         ]
+      },
+      {
+        "name": "pom-dependency-version-migration",
+        "implementation": "com.boozallen.aissemble.upgrade.migration.v1_10_0.PomDependencyVersionMigration",
+        "fileSets": [
+          {
+            "includes": ["pom.xml"]
+          }
+        ]
+      },
+      {
+        "name": "java-package-migration",
+        "implementation": "com.boozallen.aissemble.upgrade.migration.v1_10_0.JavaPackageMigration",
+        "fileSets": [
+          {
+            "includes": ["**/*.java"],
```

A: We could have this run only on classes within active modules by restricting this fileset to only `src`. That way we don't grab all `.java` files in the root execution and instead catch each in the respective execution.
```suggestion
            "includes": ["src/**/*.java"],
```

I'm also realizing that I really could have done this with the YAML migration. I was worried about Java vs Python modules, but it still could have been achieved with just a couple of filesets.

---

Updated